rust/kernel/allocator.rs

```diff
@@ -14,10 +14,14 @@ unsafe impl GlobalAlloc for KernelAllocator {
     unsafe fn alloc(&self, layout: Layout) -> *mut u8 {
         // `krealloc()` is used instead of `kmalloc()` because the latter is
         // an inline function and cannot be bound to as a result.
+        // SAFETY: FFI call to krealloc.
```

Comment:
Elsewhere we do not mention the function being called:

```suggestion
        // SAFETY: FFI call.
```

Reply:
Ok. Removed mention of the function being called.